tests/integration/dhcp_test.py

```diff
@@ -31,18 +32,25 @@
 IPV4_ADDRESS2 = '192.0.2.252'
 IPV6_ADDRESS1 = '2001:db8:1::1'
 IPV6_ADDRESS2 = '2001:db8:2::1'
+IPV4_CLASSLESS_ROUTE1 = '198.51.100.0/24'
+IPV4_CLASSLESS_ROUTE_DST1 = '192.0.2.1'
 
 DHCP_SRV_NIC = 'dhcpsrv'
 DHCP_CLI_NIC = 'dhcpcli'
+DHCP_SRV_IP4 = IPV4_ADDRESS1
+DHCP_SRV_IP6 = IPV6_ADDRESS1
 
 DNSMASQ_CONF_STR = """
 interface={}
 dhcp-range=192.0.2.200,192.0.2.250,255.255.255.0,48h
 enable-ra
 dhcp-range=2001:db8:1::100,2001:db8:1::fff,64,480h
-""".format(DHCP_SRV_NIC)
+dhcp-option=option:classless-static-route,198.51.100.0/24,192.0.2.1
```

Comment:
It seems linke you wanted to use the `IPV4_CLASSLESS` constants here.

Reply:
Done.